xbmc/addons/IAddon.h

```diff
@@ -15,6 +15,35 @@
 
 class TiXmlElement;
 
+namespace KODI
+{
+namespace ADDONS
+{
+
+/*!
+ * @brief Identifier denoting default add-on instance.
+ *
+ * All numbers greater than 0 denote add-ons with support for multiple instances.
+ */
+constexpr int32_t ADDON_SINGLETON_INSTANCE_ID = 0;
```

Comment:
Are there instance id values less than zero possible? If not, this should be unsigned -> `uint32_t`

Reply:
Yes, was gone wrong and should be uint32_t.